core/src/main/java/org/infinispan/protostream/config/AnnotationConfiguration.java

```diff
@@ -46,8 +48,10 @@ interface Builder {
 
       Builder repeatable(String containerAnnotationName);
 
+      Builder packageName(String packageName);
```

Comment:
not sure where it is used.

Reply:
These methods are used in the `IndexMetadata` class in Infinispan's query server module https://github.com/infinispan/infinispan/pull/10082/files#diff-089b23793d2cd6e9ceb648e03695fcade4d3944dedf7cfb4ea190df26764b04fR246

Reply:
got it